### ISSUE TITLE
conflicting-seller-test - b - Opportunity with price doesn't create a correct totalPaymentDue

### ISSUE THREAD

**Issue by JoshuaLevett** [openactive/openactive-test-suite#200]
A conflicting seller test at B will not correctly generate a totalPaymentDue, causing an TotalPaymentDueMismatchError to be thrown, instead of (before) the expected SellerMismatchError.
Changing conflicting-seller-test.js to only use TestOpportunityBookableFree, rather than TestOpportunityBookable, prevents this error, though this would require a system to provide free opportunities in order to pass the test

**Comment by nickevansuk** [comment]
Good point - perhaps something for the booking engine (OpenActive.Server.NET)...
